game_eggs/steamcmd_servers/sonsoftheforrest/egg-sons-of-the-forest.json

```diff
@@ -0,0 +1,154 @@
+            "container": "ghcr.io\/parkervcp\/installers:debian",
+            "entrypoint": "bash"
+        }
+    },
+    "variables": [
+        {
+            "name": "QueryPort",
+            "description": "UDP port used by Steam to list the server and enable the discovery services.",
+            "env_variable": "QUERY_PORT",
+            "default_value": "27016",
+            "user_viewable": true,
+            "user_editable": false,
+            "rules": "required|integer|between:1024,65536",
+            "field_type": "text"
+        },
+        {
+            "name": "BlobSyncPort",
+            "description": "BlobSyncPort UDP port used by the BlobSync system to initialize game systems and exchange data.",
+            "env_variable": "BLOBSYNC_PORT",
+            "default_value": "9700",
+            "user_viewable": true,
+            "user_editable": false,
+            "rules": "required|integer|between:1024,65536",
+            "field_type": "text"
+        },
+        {
+            "name": "Server Name",
+            "description": "Name of the server visible in the server list, and in the Steam contacts.",
+            "env_variable": "SRV_NAME",
+            "default_value": "Pterodactyl hosted Server",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|max:60",
```

Comment:
Not sure if there is a real limit required. The documentation only says "Any string".

Is there a specific Limit given?

Reply:
> Not sure if there is a real limit required. The documentation only says "Any string".
> 
> Is there a specific Limit given?

shgow me, that more is allowed ;)

Reply:
https://steamcommunity.com/sharedfiles/filedetails/?id=2992700419&snr=1_5_9_ Here ^^
The section says "any string"

Reply:
![image](https://github.com/parkervcp/eggs/assets/13904508/a0cd8c76-b50d-4fdb-a9f7-ae4c9882a708)